DANS-KNAW/easy-solr4files-index
# EASY-1320 commands made available as http request

Fixes another chunk of EASY-1320

#### When applied it will
* have commands made available as http request
* 
* 

#### Where should the reviewer @DANS-KNAW/easy start?

The changes in the readme which should reflect the servlet class.

#### How should this be manually tested?

* `mvn clean install && vagrant destroy -f && vagrant up`
* Check the result in your browser with `http://test.dans.knaw.nl:8983/solr/#/fileitems/query`, you should find nothing.
* `curl -X POST 'http://test.dans.knaw.nl/fileindex/init'`
  it should report one bag for the pdbs store.
* Repeat the check with the browser. Now you should find 9 file items.
* `curl -X DELETE 'http://test.dans.knaw.nl/fileindex/?q=attr_easy_dataset_store_id:pdbs'`
* Repeat the check with the browser. Now you should find nothing again.

#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
